### DISCUSSION TITLE
Extended mode stream - Error, or am I stupid?

### DISCUSSION THREAD

**Discussion by generousjj** [tweepy/tweepy#1642] [Questions] [answered]
Hello all,

For some reason, my stream isn't catching extended entities by default; it's instead only showing the non-extended entities. How do I get the extended_entities/where can I pass that I want an extended stream? I'm having this issue with both regular and async streaming.

Thanks,
JJ

**Comment by Harmon758** [comment, marked as answer]
See the [Streaming section of Tweepy's documentation for Extended Tweets](https://tweepy.readthedocs.io/en/latest/extended_tweets.html#streaming).
